interop-testing/src/main/java/io/grpc/testing/integration/XdsTestClient.java

```diff
@@ -143,6 +146,8 @@ private void parseArgs(String[] args) {
               + c.qps
               + "\n  --rpc_timeout_sec=INT  Per RPC timeout seconds. Default: "
               + c.rpcTimeoutSec
+              + "\n  --wait_for_ready_BOOL  Queue RPCs until channel is ready. Default: "
```

Comment:
What is the motivation for adding this? Just manual testing? (FYI I'm working on a test spec doc https://github.com/grpc/grpc/pull/21856 that describes these params for all languages, but I don't think wait-for-ready is necessary for any of the lb tests internal or external that I've looked at)

I think a description like "Enable wait-for-ready for outgoing RPCs" would be better, as that's a better defined behavior than just saying queuing. Relatedly, our internal massage client has a wait-for-channel-ready option that doesn't set wait for ready on any RPCs, but delays the first RPC until the channel state moves to connected, which may also be something be end up adding here.

Reply:
Also, if this is because the first few RPCs fail due to the 2 second deadline, please feel free to bump that value to something more appropriate. What would typically be enough for name resolution to complete? I was using 10 seconds for the Go client and that seemed to work well.

Reply:
Setting a longer deadline works for the issue of first few RPCs failure. But I came across another issue, which is TD might respond extremely slowly (e.g., using an invalid zone containing a dash in bootstrap), which causes XdsClient to conclude some resource was not found, and then channel turns to transient failure.

You are right, wait-for-ready RPC doesn't seem to have many use cases in tests except we want to mask that specific TD behavior mentioned above. 

Alright, just leave it here then (the option description is changed). We can come back if we want it in later.

Reply:
That makes sense, thanks. Even if it's just for manual run at the moment this looks like good option to have